web/src/components/ActivityIcon/ActivityIcon.js

```diff
@@ -0,0 +1,39 @@
+import { Icon } from '@chakra-ui/react'
+import {
+  FaBiking,
+  FaWeightHanging,
+  FaOm,
+  FaRegCalendarAlt,
+  FaRunning,
+  FaSwimmer,
+  FaTrophy,
+} from 'react-icons/fa'
+
+export const ActivityIcon = ({ activity, color = 'gray.500' }) => {
+  const activityTheme = activityThemes[activity.name]
+  return <Icon as={activityTheme.icon} color={color} />
+}
+
+const activityThemes = {
+  Cycling: {
+    icon: FaBiking,
+  },
+  Event: {
+    icon: FaRegCalendarAlt,
+  },
+  Triathlon: {
+    icon: FaTrophy,
+  },
+  Running: {
+    icon: FaRunning,
+  },
+  Strength: {
+    icon: FaWeightHanging,
+  },
+  Swimming: {
+    icon: FaSwimmer,
+  },
+  Yoga: {
+    icon: FaOm,
+  },
```

Comment:
I'm less and less convinced that these activities should be database-driven. They won't change much, and I'll need to assign a react-icon for new activities anyway. 

Long-term I'm planning on moving them out of the database into a fully-static set of options.